python-checks/src/test/resources/checks/itemOperationsTypeCheck/itemOperations_delitem.py

```diff
@@ -180,3 +180,12 @@ class ExtendedMock(MagicMock):
 
     def custom_mock():
         del ExtendedMock()[42]
+
+
+class SomeClass:
+    ...
+
+def get_state_fp():
+    some_class = SomeClass()
+    state = SomeClass.__getstate__(some_class)  # __getstate__ usually returns a dict but is annotated to return 'object' in Typeshed
```

Comment:
Unfortunately, I believe we'd need to update the annotation in Typeshed to fix this. I think we can accept the FP for now.